src/improvements/script/install-mise.sh

```diff
@@ -0,0 +1,295 @@
+#!/bin/sh
+set -eu
+
+# Sourced from https://mise.run/.
+# A local, static copy is maintained to reduce the risk of introducing a vulnerable `mise` version.
```

Comment:
Can you add more comments here about this, e.g. if we want to update mise to the latest, does the code at https://mise.run/ change and we just copy that in here?

Reply:
Yes, exactly. I'll add this in.